src/renderer/views/Hashtag/Hashtag.vue

```diff
@@ -35,11 +35,185 @@
           @keydown.space.prevent="handleFetchMore"
           @keydown.enter.prevent="handleFetchMore"
         >
-          <font-awesome-icon :icon="['fas', 'search']" /> {{ $t("Search Filters.Fetch more results") }}
+          <FontAwesomeIcon :icon="['fas', 'search']" /> {{ $t("Search Filters.Fetch more results") }}
         </div>
-      </ft-auto-load-next-page-wrapper>
-    </ft-card>
+      </FtAutoLoadNextPageWrapper>
+    </FtCard>
   </div>
 </template>
-<script src="./Hashtag.js" />
+<script setup>
+import { computed, onMounted, ref, shallowRef, watch } from 'vue'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import FtCard from '../../components/ft-card/ft-card.vue'
+import FtElementList from '../../components/FtElementList/FtElementList.vue'
+import FtFlexBox from '../../components/ft-flex-box/ft-flex-box.vue'
+import FtLoader from '../../components/ft-loader/ft-loader.vue'
+import FtAutoLoadNextPageWrapper from '../../components/ft-auto-load-next-page-wrapper/ft-auto-load-next-page-wrapper.vue'
+import store from '../../store/index'
+import { useRoute } from 'vue-router/composables'
+import packageDetails from '../../../../package.json'
+import { getHashtagLocal, parseLocalListVideo } from '../../helpers/api/local'
+import { copyToClipboard, setPublishedTimestampsInvidious, showToast } from '../../helpers/utils'
+import { isNullOrEmpty } from '../../helpers/strings'
+import { getHashtagInvidious } from '../../helpers/api/invidious'
+import { useI18n } from '../../composables/use-i18n-polyfill'
+const { t } = useI18n()
+
+const route = useRoute()
+
+const hashtag = ref('')
+const hashtagContinuationData = ref(null)
```

Comment:
As mentioned in the Matrix chat, Vue 3's reactivity breaks YouTube.js and as we only need to know when someone assigns something to `.value =`, we don't need deep reactivity here.
```suggestion
const hashtagContinuationData = shallowRef(null)
```